### DISCUSSION TITLE
R encountered a fatal error when fitting the example model on Mac

### DISCUSSION THREAD

**Discussion by Feiyan-Liu** [nlmixr2/nlmixr2#41] [General]
R version 4.1.2 (2021-11-01) -- "Bird Hippie"
Copyright (C) 2021 The R Foundation for Statistical Computing
Platform: x86_64-apple-darwin17.0 (64-bit)

After installing the RxODE2/nlmixr2 packages, I tested the example model from tutorial however it was always terminated with the error message saying "R encountered a fatal error". 
I tried two estimate methods, "saem" and "foce', same problems occurred.
Dose anyone have any experiences or suggestions on this?

**Comment by mattfidler** [comment]
There are some mac architectures that do not work well for `symengine`

Could you try:

```r
library(symengine)
```

And see what happens.  By the way, I converted this to a discussion.

**Reply by Feiyan-Liu** [reply]
Thanks for your suggestion, however my R still crushed :(

**Reply by mattfidler** [reply]
Did it crash when loading symengine?

**Reply by Feiyan-Liu** [reply]
it worked fine when loading symengine but crushed before finishing the fitting process

**Reply by mattfidler** [reply]
Since I cannot reproduce your error I can't help any more, nor can I pinpoint what the problem is without more information either.

A possibility is to add a reproducible example (some use the `reprex()` package).  If the model crashes on the systems I have access to (pc and linux) then I can try to help.  Otherwise, I cannot help you any further.

**Reply by Feiyan-Liu** [reply]
I'm so appreciate your kind reply and your time. 
Actually what I did is (1) installed the packages and (2) run the exact basic example code, then it crushed. But when I later tested it on a Windows pc it worked well, so it might be something related to Mac, or my Mac. 
I'll try to update my R version and see if it will make any differences.
Thanks.

**Reply by Feiyan-Liu** [reply]
It works well after I updated my R to version 4.2.1 !

**Reply by mattfidler** [reply]
That is great news.  Thanks for sharing what worked for you.
